src/features/projectsV2/ProjectsContext.tsx

```diff
@@ -406,6 +410,8 @@ export const ProjectsProvider: FC<ProjectsProviderProps> = ({
       setSelectedSamplePlantLocation,
       selectedSite,
       setSelectedSite,
+      preventShallowPush,
```

Comment:
Since `preventShallowPush` is only accessed in the context, don't pass it on as a part of `value`